packages/experimental/ExperimentalIcon/SPEC.md

```diff
@@ -1,5 +1,171 @@
 # Icon
 
-Write smth here
+Icon component contains SvgIcon and FontIcon components.
```

Comment:
Will migration documentation be its own PR?

Reply:
Yes